src/en/asurascans/src/eu/kanade/tachiyomi/extension/en/asurascans/AsuraScans.kt

```diff
@@ -154,8 +154,11 @@ class AsuraScans : ParsedHttpSource(), ConfigurableSource {
     }
 
     private fun getGenreFilters(): List<Genre> = genresList.map { Genre(it.first, it.second) }
-    private fun getStatusFilters(): List<Pair<String, String>> = statusesList.map { it.first to it.second.toString() }
-    private fun getTypeFilters(): List<Pair<String, String>> = typesList.map { it.first to it.second.toString() }
+    private fun getStatusFilters(): List<Pair<String, String>> =
+        statusesList.map { it.first to it.second.toString() }
+
+    private fun getTypeFilters(): List<Pair<String, String>> =
+        typesList.map { it.first to it.second.toString() }
```

Comment:
Don't change the style of unrelated code unless the linter complains. It just adds noise to the diff.